### PR TITLE
DEVEXP-550_Single cucumber statement for conversation webhooks events

### PR DESCRIPTION
**Pull request by asein-sinch**
The E2E tests will pass when the corresponding PR will be merged on the mockserver side

### CODE REVIEW

**Review by Dovchik — COMMENTED** [review]
Depends on how our decision of handling scenario outline or not
https://github.com/sinch/sinch-sdk-mockserver/pull/38#pullrequestreview-2276938050

File: packages/conversation/tests/rest/v1/webhooks-events/webhooks-events.steps.ts [hunk 1]
```diff
@@ -15,23 +15,22 @@ const processEvent = async (response: Response) => {
...
-Then('the Conversation event header contains a valid signature', () => {
-  assert.ok(conversationCallbackWebhook.validateAuthenticationHeader(formattedHeaders, rawEvent));
-});
-
 When('I send a request to trigger a "CAPABILITY" event', async () => {
   const response = await fetch('http://localhost:3014/webhooks/conversation/capability-lookup');
   await processEvent(response);
 });
 
+Then('the header of the Conversation event "CAPABILITY" contains a valid signature', () => {
```
Comment (JPPortier) [inline packages/conversation/tests/rest/v1/webhooks-events/webhooks-events.steps.ts:30]: Is _just_ writing `Then('the header of the Conversation event {string} contains a valid signature'` could not help to have a single `Then` for all cases within this file ?

**Review by JPPortier — APPROVED** [review]

File: packages/conversation/tests/rest/v1/webhooks-events/webhooks-events.steps.ts (hunk 1 above)
Comment (asein-sinch) [reply packages/conversation/tests/rest/v1/webhooks-events/webhooks-events.steps.ts:30]: On my side, no problem, but if you run the tests multithreaded, you'll need to know with header you are validating

Comment (JPPortier) [reply packages/conversation/tests/rest/v1/webhooks-events/webhooks-events.steps.ts:30]: This is handled: https://github.com/sinch/sinch-sdk-java/pull/145/files#diff-39a2d8af5bc307b3fc35d03b23fa7639deae0d13f5c0411dba24c6c063ca600fR230
Receiving the `String e2eKeyword` parameter, java e2e tests are able to get event related to keyword

In current Node context: parameter won't be used but you could avoid code duplication

Comment (asein-sinch) [reply packages/conversation/tests/rest/v1/webhooks-events/webhooks-events.steps.ts]: Thanks for the information, I've removed the duplicated code on my side.

**Pull request merged**